include/cosim/timer.hpp

```diff
@@ -36,6 +36,15 @@ struct real_time_config
      * This value is used for monitoring purposes only.
      */
     std::atomic<int> steps_to_monitor = 5;
+
+    /**
+     *  A sampling period in second(s) used in the rolling average real time factor calculation.
```

Comment:
Perhaps you should explicitly state that the sampling period is specified in real (wall clock) time rather than logical (simulation) time, to avoid confusion.

Reply:
Updated the comment